src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './index.css';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => (
+  <nav className="navbar">
+    <h2 className="navbar__title">RentIt</h2>
+    <ul className="navbar__links">
+      <li><NavLink to="cars" activeClassName="active">Cars</NavLink></li>
```

Comment:
- A small part of the navbar title's box model is intersecting with the nav link `Cars` as shown in the image below. making that small area not clickable.
![fix](https://user-images.githubusercontent.com/88746054/229339863-77de2e9e-5c6d-44fa-9de9-8186ad0b415e.png)

Reply:
Done, thank you